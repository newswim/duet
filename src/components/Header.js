import React from 'react'
import {
  Container,
  HeaderContainer,
  BigHeading,
  HeaderDescription,
  HeaderImage,
  Link
} from '../styles'

const Header = props => {
  return (
    <HeaderContainer>
      <Container>
        <HeaderImage src='/static/img/logo-notext.png' width={120} height={120} />
        <BigHeading>{props.title}</BigHeading>
        <HeaderDescription>
          {props.description}
        </HeaderDescription>
      </Container>
    </HeaderContainer>
  )
}

export default Header
