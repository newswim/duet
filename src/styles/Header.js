import styled from 'styled-components'

export const Heading = styled.h1`
  font-size: 24px;
`

export const Small = styled.span`
  font-size: 16px;
  font-weight: normal;
`
