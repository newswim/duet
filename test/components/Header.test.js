import React from 'react'
import Header from '../../src/components/Header'
import renderer from 'react-test-renderer'

test('it renders the header structure', () => {
  const tree = renderer.create(
    <Header />
  )

  expect(tree).toMatchSnapshot()
})
