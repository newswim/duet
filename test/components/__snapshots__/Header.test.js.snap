// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the header structure 1`] = `
<div
  className="bkMRIm"
>
  <div
    className="erWBAn"
  >
    <img
      className="gfQWxQ"
      height={120}
      src="/static/img/logo-notext.png"
      width={120}
    />
    <h1
      className="Jpfkj"
    >
      Welcome to duet!
    </h1>
    <p
      className="dlsGdS"
    >
      duet is a tool that helps newcomers to open-source find their
      <a
        className="jnqCsK"
        href="https://github.com/jekyll/community/blob/master/policies/first_timers_only.md"
      >
        Â first contribution to Jekyll
      </a>
      , and to help maintainers keep an overview.
    </p>
  </div>
</div>
`;
